packages/integrations/solid/README.md

```diff
@@ -98,6 +98,41 @@ export default defineConfig({
 });
 ```
 
+## Async rendering strategy
```

Comment:
@patdx Making a note that this change should now be PR'd to the docs repo directly!

We have updated all the integration READMEs to a minimal format, so instead, please make a PR in the docs repo for this content change at: https://github.com/withastro/docs/blob/main/src/content/docs/en/guides/integrations-guide/solid-js.mdx and do not edit the Solid-js README. Thank you!

Reply:
Got it, I made a PR for the docs here: https://github.com/withastro/docs/pull/6027

Reply:
Thanks, got it! We'll pick this part of the conversation up over there! :partying_face: